src/main/java/com/robosh/ejournal/data/repository/ValidationRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.robosh.ejournal.data.repository;
+
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ValidationRepository {
```

Comment:
Why it has to be repository, when it's an util class?
Or am I missing something?

Reply:
I have already removed it)